photutils/psf/building/utils.py

```diff
@@ -0,0 +1,106 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Utility functions for PSF building.
+"""
+from __future__ import absolute_import, division, print_function
+
+import numpy as np
+
+__all__ = ['py2round', 'interpolate_missing_data']
+
+
+def py2round(x):
+    """
+    This function returns a rounded up value of the argument, similar
+    to Python 2.
+
+    """
+    if hasattr(x, '__iter__'):
+        rx = np.empty_like(x)
+        m = x >= 0.0
+        rx[m] = np.floor(x[m] + 0.5)
+        m = np.logical_not(m)
+        rx[m] = np.ceil(x[m] - 0.5)
+        return rx
+
+    else:
+        if x >= 0.0:
+            return np.floor(x + 0.5)
+        else:
+            return np.ceil(x - 0.5)
+
+
+def interpolate_missing_data(data, method, mask=None, const_fillval=0.0):
```

Comment:
@larrybradley , if `interpolate_masked_data` is removed, it will no longer be available to anyone who is relying on it (not in `photutils` but in their own codes). Are you sure?